src/main/java/com/sourcegraph/cody/agent/CodyAgentCodebase.kt

```diff
@@ -39,9 +40,9 @@ class CodyAgentCodebase(private val underlying: CodyAgentServer, val project: Pr
   }
 
   private fun onRepositoryName(repositoryName: String?) {
```

Comment:
nit: can we rename it to `onRepositoryNameChange`

Reply:
fixed